frontend/src/auth/AuthProvider.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+import { useContext, useState } from 'react'
+import jwt from 'jsonwebtoken'
+import AuthContext from './AuthContext'
+
+export default function AuthProvider({ children }) {
+  const [token, setToken] = useState()
+
+  const login = credentials =>
+    axios
+      .post('auth/access_token', credentials)
+      .then(response => response.data)
+      .then(dto => dto.token)
+      .then(token => setToken(token))
+      .catch(error => console.error(error))
```

Comment:
die methode sollte in den service. hier musst du auch keinen catch machen da du den error dort handeln musst wo du die login methode aufrufst... also auf der loginpage